compiler/src/iree/compiler/Dialect/Flow/Conversion/TensorToFlow/Patterns.cpp

```diff
@@ -183,6 +183,59 @@ struct ConvertTensorFromElementsPattern
   }
 };
 
+/// Populates given sizes array with the dynamic dims.
+static SmallVector<Value> sizesForTensor(OpBuilder &builder, Location loc,
```

Comment:
nit: rename to `getDynamicTensorSizes` to make it clearer it is specifically getting dynamic sizes.